app/namespace.go

```diff
@@ -1312,60 +1405,125 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				},
 			},
 			{
-				Name:    "delete",
-				Aliases: []string{"d"},
-				Usage:   "Delete export sink",
+				Name:    "update",
+				Aliases: []string{"u"},
+				Usage:   "Update export sink",
 				Flags: []cli.Flag{
 					NamespaceFlag,
 					sinkNameFlag,
+					sinkEnabledFlag,
+					sinkAssumedRoleFlagOptional,
+					s3BucketFlagOptional,
 					ResourceVersionFlag,
+					kmsArnFlag,
 					RequestIDFlag,
 				},
 				Action: func(ctx *cli.Context) error {
 					namespaceName := ctx.String(NamespaceFlagName)
 					sinkName := ctx.String(sinkNameFlag.Name)
-					resourceVersion, err := c.getExportSinkResourceVersion(ctx, namespaceName, sinkName)
+					sink, err := c.getExportSink(ctx, namespaceName, sinkName)
 					if err != nil {
 						return err
 					}
+					resourceVersion := c.selectExportSinkResourceVersion(ctx, sink)
+
+					isEnabledChange, err := c.isSinkEnabledChange(ctx, sink)
+					if err != nil {
+						return err
+					}
+
+					if !isEnabledChange && !c.isAssumedRoleChange(ctx, sink) && !c.isKmsArnChange(ctx, sink) && !c.isS3BucketChange(ctx, sink) {
+						fmt.Println("nothing to update")
+						return nil
+					}
+
+					if isEnabledChange {
```

Comment:
the behavior here suggests the flag is more of a "toggle" than an "enable?" Can we rename the flag and its usage to make this more clear?